tests/test_auto_detection.py

```diff
@@ -37,19 +40,25 @@ def test_loop_auto():
     assert isinstance(policy, asyncio.events.BaseDefaultEventLoopPolicy)
     expected_loop = "asyncio" if uvloop is None else "uvloop"
     assert type(policy).__module__.startswith(expected_loop)
-    loop = asyncio.get_event_loop()
-    loop.close()
+
+    async def get_loop():
+        return anyio.lowlevel.current_token()
+
+    loop = anyio.run(get_loop)
+    assert loop.__module__.startswith(expected_loop)
```

Comment:
I dont know anyio at all, what does this do and why introducing it, also if kept shouldn't it be added to the requirements.txt ?

Reply:
it's a dep via httpx, it's a handy shim for `get_running_loop()` on py3.6-py3.10

Reply:
it's to save writing:

```python
if sys.version_info >= (3, 7):
    async def get_loop():
        return asyncio.get_running_loop()

    loop = asyncio.run(get_loop())
else:

   with contextlib.closing(asyncio.get_event_loop()) as loop:
       pass
```